AndroidNativeKotlinTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -1,55 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-	android:versionCode="1"
-	android:versionName="1.0"
-	android:installLocation="internalOnly">
+    android:installLocation="internalOnly"
+    android:versionCode="1"
+    android:versionName="1.0">
 
-	<application android:icon="@drawable/sf__icon"
-	    android:label="@string/app_name"
-		android:name=".MainApplication"
-		android:manageSpaceActivity="com.salesforce.androidsdk.ui.ManageSpaceActivity">
+    <application
+        android:name=".MainApplication"
+        android:icon="@drawable/sf__icon"
+        android:label="@string/app_name"
+        android:manageSpaceActivity="com.salesforce.androidsdk.ui.ManageSpaceActivity">
 
-		<!-- Launcher screen -->
-		<activity android:name=".MainActivity"
-		    android:label="@string/app_name"
-			android:theme="@style/SalesforceSDK"
-            android:exported="true">
-			<intent-filter>
-				<action android:name="android.intent.action.MAIN" />
-				<category android:name="android.intent.category.LAUNCHER" />
-			</intent-filter>
-		</activity>
+        <!-- Launcher screen -->
+        <activity
+            android:name=".MainActivity"
+            android:exported="true"
+            android:theme="@style/SalesforceSDK">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
 
         <!-- Login activity -->
-        <!--
-            To enable browser based authentication, uncomment the lines below and replace
-            'scheme', 'host' and 'path' with their corresponding values from your connected app.
-
-            For example, if the callback URL of your connected app is
-            "testsfdc:///mobilesdk/detect/oauth/done",
-            'scheme' would be "testsfdc", 'host' would be "*" since it doesn't exist, and
-            'path' would be "/mobilesdk/detect/oauth/done".
-
-            If the callback URL is "sfdc://login.salesforce.com/oauth/done",
-            'scheme' would be "sfdc", 'host' would be "login.salesforce.com",
-            and 'path' would be "/oauth/done".
-        -->
-        <!--
-        <activity android:name="com.salesforce.androidsdk.ui.LoginActivity"
-            android:theme="@style/SalesforceSDK"
```

Comment:
I don't think we want to remove this.

Reply:
That's interesting.  I didn't realize that was also commented out on `dev`.  Should we also bring back the comment block as well? @wmathurin had removed it during his spike.

Reply:
@brandonpage - This should be resolved for you in another comment thread.